packages/engine/Source/Scene/Picking.js

```diff
@@ -301,6 +314,82 @@ Picking.prototype.pick = function (scene, windowPosition, width, height) {
   return object;
 };
 
+/**
+ * Returns an object with information about the voxel sample rendered at
+ * a particular window coordinate. Returns <code>undefined</code> if there is no
+ * voxel at that position.
+ *
+ * @param {Scene} scene
+ * @param {Cartesian2} windowPosition Window coordinates to perform picking on.
+ * @param {number} [width=3] Width of the pick rectangle.
+ * @param {number} [height=3] Height of the pick rectangle.
+ * @returns {object} Object containing the picked primitive.
+ */
+Picking.prototype.pickVoxel = function (scene, windowPosition, width, height) {
+  //>>includeStart('debug', pragmas.debug);
+  if (!defined(windowPosition)) {
+    throw new DeveloperError("windowPosition is undefined.");
```

Comment:
Why not `Check.defined`?

Reply:
Good point. This was a copy-paste from `Picking.prototype.pick`. I switched to `Check.defined` in both places.